docs/_docs/install/server.md

```diff
@@ -90,27 +90,207 @@ We will discuss setting up https in a later section.
 
 ## Storage
```

Comment:
You are right - we possibly don't need this. Could you test for me if you change the container to standard nginx (in docker-compose) whether everything still works as it should? I was hesitant to remove the section in case there are any users that still use the older one. What I'll do for now is move it to the bottom of that page, and update it so that it reads past tense.

Reply:
Actually scratch this - I think it's still used for the click and upload (form) portion. Let's leave as is.